init.sql

```diff
@@ -18,6 +18,16 @@ CREATE TABLE `Users` (
  PRIMARY KEY (`id`)
 );
 
+CREATE TABLE `Ratings` (
+`id` int(11) NOT NULL AUTO_INCREMENT,
+`user` int(11) DEFAULT NULL,
+`movie` int(11) DEFAULT NULL,
+`rating` int(11) DEFAULT NULL,
```

Comment:
i mean... its not like that is a great naming convention... but we dont have that much of a choice